src/app/components/Labels/LabelColors.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Icon, Input } from 'react-components';
+import { LABEL_COLORS } from 'proton-shared/lib/constants';
+
+function LabelColors({ selected, onClick, className }) {
+    const getClass = (className, more) => {
+        return [className, more].filter(Boolean).join(' ');
+    };
+    return (
+        <ul className={getClass('LabelColors-container unstyled', className)}>
+            {LABEL_COLORS.map((color, i) => {
+                return (
+                    <li
+                        onClick={onClick(color)}
+                        key={'mykey' + i}
+                        className={getClass('LabelColors-item', selected === color && 'selected')}
+                        style={{ color }}
+                    >
+                        <Input
```

Comment:
Why not using `onChange` on this input radio instead of `onClick` on the `<li>`?

Reply:
that's a good question :thinking: